<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>CustomersWidget</class>
 <widget class="QWidget" name="CustomersWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>684</width>
    <height>549</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_3">
   <item>
    <widget class="QTabWidget" name="tabWidget">
     <property name="currentIndex">
      <number>1</number>
     </property>
     <widget class="QWidget" name="tab">
      <property name="autoFillBackground">
       <bool>false</bool>
      </property>
      <attribute name="title">
       <string>Admin</string>
      </attribute>
      <widget class="AdminWidget" name="widget" native="true">
       <property name="geometry">
        <rect>
         <x>0</x>
         <y>0</y>
         <width>661</width>
         <height>491</height>
        </rect>
       </property>
      </widget>
     </widget>
     <widget class="QWidget" name="tab_2">
      <attribute name="title">
       <string>Customers</string>
      </attribute>
      <widget class="QSplitter" name="splitter">
       <property name="geometry">
        <rect>
         <x>0</x>
         <y>0</y>
         <width>661</width>
         <height>491</height>
        </rect>
       </property>
       <property name="orientation">
        <enum>Qt::Orientation::Horizontal</enum>
       </property>
       <widget class="QListWidget" name="userList"/>
       <widget class="QWidget" name="EnrollWidget_3" native="true">
        <layout class="QVBoxLayout" name="verticalLayout">
         <item>
          <widget class="QLabel" name="label_3">
           <property name="sizePolicy">
            <sizepolicy hsizetype="Preferred" vsizetype="Minimum">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <property name="minimumSize">
            <size>
             <width>0</width>
             <height>50</height>
            </size>
           </property>
           <property name="maximumSize">
            <size>
             <width>16777215</width>
             <height>20</height>
            </size>
           </property>
           <property name="text">
            <string>고객 검색</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QWidget" name="EditWidget_3" native="true">
           <property name="sizePolicy">
            <sizepolicy hsizetype="Preferred" vsizetype="Maximum">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <property name="minimumSize">
            <size>
             <width>0</width>
             <height>100</height>
            </size>
           </property>
           <property name="maximumSize">
            <size>
             <width>16777215</width>
             <height>100</height>
            </size>
           </property>
           <layout class="QVBoxLayout" name="verticalLayout_2">
            <item>
             <layout class="QFormLayout" name="formLayout_3">
              <property name="fieldGrowthPolicy">
               <enum>QFormLayout::FieldGrowthPolicy::FieldsStayAtSizeHint</enum>
              </property>
              <item row="0" column="0">
               <widget class="QLabel" name="nameLabel_3">
                <property name="text">
                 <string>이름</string>
                </property>
               </widget>
              </item>
              <item row="0" column="1">
               <widget class="QTextEdit" name="userNameSearchText"/>
              </item>
             </layout>
            </item>
            <item>
             <widget class="QPushButton" name="userSearchButton">
              <property name="text">
               <string>검색</string>
              </property>
             </widget>
            </item>
           </layout>
          </widget>
         </item>
        </layout>
       </widget>
      </widget>
     </widget>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>AdminWidget</class>
   <extends>QWidget</extends>
   <header location="global">adminwidget.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
